### ISSUE TITLE
New UI

### ISSUE THREAD

**Issue by maxcnunes** [sqlectron/sqlectron-gui#546]
I would like to change the app to use a theme that looks less like a web app, if we could get something like the Slack UI it would be awesome. Also, I would like to change a few other things, like:
* Support dark mode
* Add a "struct" screen which would show details like columns and indexes about a table (instead of having it nested in the sidebar like it is nowadays)
* Fix the sidebar, similar to Slack's sidebar. Nowadays it isn't attached to the left which uses extra space unnecessary.
* Fix the look of the table results, the format, and alignment of the columns and rows have always been too buggy
* Simplify the main page and allow to filter by tags and client types
* Possibly run a single query per execution, it would simplify a lot our codebase and the UI organization. I think it was a mistake, trying to support multiple queries per execution :( But if it is something we too many people like, then maybe we could show each result in tabs.

So, initially, I need to start working on the new UI theme and components. Let me know if anyone has any existing UI themes and components you like to suggest. Otherwise, If I cannot find something nice either, I may write something from scratch. A few UI kits I have found so far:

- https://github.com/segmentio/evergreen
- https://rsuitejs.com/
- https://ant.design/

But I am not 100% happy with any of them, I am afraid using a framework like that could get us stuck with it and turn it hard to maintain in the future if we need to do anything more advanced.

Also, please suggest any other UI improvements you would like to see in the app. Not saying I will implement them, but I will definitely take them into consideration.

**Comment by MasterOdin** [comment]
> Possibly run a single query per execution, it would simplify a lot our codebase and the UI organization. I think it was a mistake, trying to support multiple queries per execution :( But if it is something we too many people like, then maybe we could show each result in tabs.

I am not sure I follow what you mean with this. Are you saying that given a workspace with multiple queries, if you hit the execute button, then it will only ever execute the first query, an that all subsequent queries would be ignored? Or just that if you had multiple select queries, you would only get the last result?

**Comment by MasterOdin** [comment]
> Support dark mode

I looked at this recently, and was wondering if you had a psd of the logo, and so as a starting point, can look to modify it to be "dark mode compatible" and then go from there with theming color wise.

**Comment by maxcnunes** [comment]
Actually, I realized there are other DB clients that support executing multiple queries at once, I think we can keep that. But I am wondering if showing the results in tabs could keep the UI cleaner.
I can check with my brother, who created the logo, if he still has the PSD file for that.

**Comment by brendanjones** [comment]
+1 to dark mode support for MacOS. Just started using SQLectron and it's great except for the white burning my retinas (I jest, I'll cope, but dark mode would be nice).

**Comment by color-space** [comment]
Can we remove the startup splashscreen? 
It feels like there is a fake timeout to wait for the animation to complete. 
Please remove this, the animation is unnecessary

**Comment by maxcnunes** [comment]
@color-space I agree that animation is annoying, but while anyone doesn't remove it, you should be able to disable it through the configuration page. Otherwise, you can disable it by changing the [configuration file](https://github.com/sqlectron/sqlectron-gui/blob/main/docs/app/configuration-file.md) directly by adding `"disabledOpenAnimation": true` to it.
